Cargo.toml

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Please remove the Nvidia copyright.

Reply:
I will raise this internally here. It would be helpful if there were some contributing guide I could refer to with the policy for this type of thing for this project. I noticed that the [LICENSE](https://github.com/substrait-io/substrait/blob/main/LICENSE) in the main substrait repo has a Dremio copyright so it isn't clear when it is appropriate to have company copyrights or not.

Reply:
Goodness, that was a mistake! I guess that is what you get when doing a copy and a paste.

Yes, a contributors guide would be good. Right now there should be no copyright notices from any corporation in the Substrait repos. Will post a patch to remove the Dremio thing... ugh.

Reply:
Posted https://github.com/substrait-io/substrait/pull/187 to fix

Reply:
Heh, I was wondering about that Dremio copyright notice yesterday too when I was fighting git to move the validator to its own repo.

Either way... "all rights reserved" makes no sense on an open source repository regardless of whose name is on it. It means no one can even look at the file, much less modify it, without explicit permission from the author stating otherwise.

Reply:
@jacques-n I am still working on resolving this copyright issue. The LICENSE file in this repo still states:

```
   APPENDIX: How to apply the Apache License to your work.

      To apply the Apache License to your work, attach the following
      boilerplate notice, with the fields enclosed by brackets "[]"
      replaced with your own identifying information. (Don't include
      the brackets!)  The text should be enclosed in the appropriate
      comment syntax for the file format. We also recommend that a
      file or class name and description of purpose be included on the
      same "printed page" as the copyright notice for easier
      identification within third-party archives.

   Copyright [yyyy] [name of copyright owner]

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

I know you had removed this section from the version of the license in the main repo so I assume you are planning on removing it here as well?

Reply:
Also, I assume it is sufficient just to add `// SPDX-License-Identifier: Apache-2.0` to the files being submitted? I think you had already confirmed this but I cannot find the conversation so just want to make sure this is correct.

Reply:
Fixed the license text and yes, SPDX license.